src/material/button-toggle/button-toggle.ts

```diff
@@ -39,9 +39,6 @@ import {
 } from '@angular/material/core';
 
 
-/** Acceptable types for a button toggle. */
-export type ToggleType = 'checkbox' | 'radio';
```

Comment:
Even though this isn't used by us anymore, it still has to go through a deprecation period, because it's exposed as a public API. Usually the way we do it is by adding an `@deprecated` message and setting the breaking using `@breaking-change 11.0.0`.

Reply:
Done. FWIW, this type appears to only ever have been used the `_type` property below, which was `@internal` at its inception and then was cut over to be private. Still, I agree that in theory someone could have imported this type and used it for completely unrelated reasons and should perhaps be subject to deprecation based on that.

Reply:
Sorry, I did a rebase/amend commit instead of cutting a new one and I think I broke the code review tool. I'll try to use full commits in the future at least during review.